agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1510,6 +1510,9 @@ void ap_manager_thread::handle_hostapd_attached()
 {
     LOG(DEBUG) << "handling enabled hostapd";
 
+    //the supported channels must always be read
```

Comment:
you can remove this comment since the code is self explained

Reply:
Removed self-explanatory unneeded comments